src/Components/DapperScrollbars/Documentation/DapperScrollbars.stories.tsx

```diff
@@ -1,72 +1,135 @@
 // Libraries
 import * as React from 'react'
-import marked from 'marked'
+// import marked from 'marked'
```

Comment:
✂️

Reply:
This story does not work in master branch. "str.replace is not a function" and I spent a lot of time trying to figure out why to no avail. I wasn't sure if we should go back and fix this, as someone else might have better insight on how to to fix it.

The commented code sets up a template for how the story should look, but doesn't currently work as mentioned above. The comments allow the story to work but removes the templating.

Ok, I will go ahead and remove the commented code entirely.